pkg/minishift/config/allinstancesconfig.go

```diff
@@ -24,10 +24,16 @@ import (
 
 var AllInstancesConfig *GlobalConfigType
 
+type Profile struct {
```

Comment:
only active profile seems to be needed in my opinion.
and if that one does not exist, you fall back to a default... and if that does not exist, there is no active in use.

Reply:
@gbraad The idea is to check the actual files to decide what profiles are present. At this point all instance config is used for it. If an user has defined a profile we will not delete it till user runs `minishift delete --all`

Reply:
> only active profile seems to be needed in my opinion.

that was mu suggestions as well. Less house keeping and less chances to get out of sync.